lua/nvim-lsp-installer/servers/fsautocomplete/init.lua

```diff
@@ -0,0 +1,27 @@
+local server = require "nvim-lsp-installer.server"
+local path = require "nvim-lsp-installer.path"
+local std = require "nvim-lsp-installer.installers.std"
+local shell = require "nvim-lsp-installer.installers.shell"
+
+return function(name, root_dir)
+    return server.Server:new {
+        name = name,
+        root_dir = root_dir,
+        languages = { "fsharp" },
+        homepage = "https://fsharp.org",
+        installer = {
+            std.ensure_executables {
+                { "dotnet", "dotnet was not found in path. fsautocomplete is installed as global dotnet tool." },
+            },
+            shell.polyshell[[dotnet tool update --global fsautocomplete]],
+            shell.polyshell[[dotnet tool update --global fantomas-tool --version "*-*"]],
+        },
+        default_options = {
+            cmd = { "dotnet", "fsautocomplete", "--background-service-enabled" },
+            filetypes = { "fsharp" },
+            init_options = {
+               AutomaticWorkspaceInit = true,
+            },
```

Comment:
These options should not be needed - they should be governed by lspconfig, and I can see they [already exist there](https://github.com/neovim/nvim-lspconfig/blob/master/lua/lspconfig/fsautocomplete.lua#L10-L13)! So this can be removed

```suggestion
```

Reply:
Removed it from this PR. It needs some changes. I will make PR in lspconfig.